app/models/user.rb

```diff
@@ -984,7 +988,7 @@ def editable_fields_of_user(user)
 
   private def editable_competitor_info_fields(user)
     fields = Set.new
-    if user == self || admin? || any_kind_of_delegate? || results_team? || communication_team?
+    if user == self || can_edit_any_user?
```

Comment:
Bit weird to have an unless immediately after an if block - I suggest merging them so that the line is:

```ruby
if user == self || can_edit_any_user? || !cannot_edit_data_reason_html(user)
```

Reply:
What do you mean by that?

Reply:
Daniel is right, you cannot merge anything here. The GitHub diff view might be misleading: There's actually stuff happening inside the outer `if ...` block but _outside_ of the nested `unless` block.

Reply:
Ah sorry I see now! Probably just me misreading the diff.